earn/src/data/constants/Values.ts

```diff
@@ -27,3 +27,8 @@ export const API_PRICE_RELAY_LATEST_URL = 'https://api-price.aloe.capital/price-
 export const API_PRICE_RELAY_HISTORICAL_URL = 'https://api-price.aloe.capital/price-relay/v1/historical';
 export const API_PRICE_RELAY_CONSOLIDATED_URL = 'https://api-price.aloe.capital/price-relay/v1/consolidated';
 export const API_REDEEM_REWARD_URL = 'https://api-claim.aloe.capital/v1/claim';
+
+export function primeUrl() {
+  // NOTE: trailing `/` is important for .eth domain resolution to work
+  return isDappnet() ? 'https://prime.aloe.eth/' : 'https://prime.aloe.capital/';
+}
```

Comment:
I don't have a strong opinion, just put it here because it seemed easier to keep everything in one place (instead of jumping between Values.ts and Utils.ts and potentially having to import from both of them too).